src/actions/market.js

```diff
@@ -123,8 +119,8 @@ export const requestMarketTradesForAccount = (marketAddress, accountAddress) =>
  * Requests all trades (tradehistory) on a market from GnosisDB.
  * @param {Market} market
  */
-export const requestMarketTrades = marketAddress => async (dispatch) => {
-  const payload = await api.requestMarketTrades(marketAddress)
+export const requestMarketTrades = market => async (dispatch) => {
```

Comment:
https://github.com/gnosis/olympia-interface/pull/173/files#diff-a7e638fe149ee1e4fb43773cc5f14672R122
This change is from my PR which is still in testing 😮

Reply:
It's okay, I talked to Lama, it works 👍